apps/account-api/src/api.module.ts

```diff
@@ -23,6 +25,7 @@ import apiConfig from './api.config';
 import { APP_FILTER } from '@nestjs/core';
 import { AllExceptionsFilter } from '#utils/filters/exceptions.filter';
 import { QueueModule } from '#queue/queue.module';
+import { getPinoHttpOptions } from '../../../libs/logger/logLevel-common-config';
```

Comment:
nit: it would be nice to create a module alias like `#logger`, like we did for all of the other modules. Relative-path imports like this are ugly, IMO

Reply:
done